features/admin.applications.v1/components/forms/inbound-oidc-form.tsx

```diff
@@ -1609,26 +1609,29 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
                                     }
                                     skipInternalValidation= {
                                         template.templateId === ApplicationManagementConstants.MOBILE
+                                        || CustomApplicationTemplate?.id === template?.id
                                     }
                                     validation={ (value: string) => {
-                                        if (
-                                            !(isMobileApplication)
-                                            && CustomApplicationTemplate?.id !== template?.id
-                                            && !(URLUtils.isURLValid(value, true) &&
-                                                (URLUtils.isHttpUrl(value)
-                                                || URLUtils.isHttpsUrl(value)))
-                                        ) {
+                                        if ((isMobileApplication || CustomApplicationTemplate?.id === template?.id)) {
+                                            if (URLUtils.isMobileDeepLink(value)) {
+                                                setCallbackURLsErrorLabel(null);
+
+                                                return true;
+                                            }
 
                                             return false;
                                         }
```

Comment:
If the app is a custom app (standard oidc app) and user entered a url that is not a valid mobile deep link, but a valid http url, this block will be evaluated to false, won't it?

Reply:
`isMobileDeepLink` regex also matches http and https URLs. Deep links can be http, https or custom URI scheme links..